devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy

```diff
@@ -19,6 +19,8 @@ metadata {
 		capability "Polling"
 		capability "Refresh"
 		capability "Sensor"
+		capability "Configuration"
```

Comment:
Actually, I would prefer using updated() in *all* cases for setting checkInterval, since it is called after changing a device's type (manually via the web interface). At least when it's not tied to specific configuration commands.

cc @jackchi